.coffeelintignore

```diff
@@ -1,2 +0,0 @@
-iconsets
-node_modules
```

Comment:
Wow... That script is news to me. So the `--format tap` flag I added to the package.json only runs locally.